web-common/src/main/java/edu/unc/lib/boxc/web/common/services/AccessCopiesService.java

```diff
@@ -6,6 +6,7 @@
 import edu.unc.lib.boxc.model.api.DatastreamType;
 import edu.unc.lib.boxc.model.api.ResourceType;
 import edu.unc.lib.boxc.model.api.ids.PID;
+import edu.unc.lib.boxc.model.api.objects.WorkObject;
```

Comment:
Seems like we can roll back the changes to this file, since it's just adding an unused import